src/app/store/slices/sharedLinks/index.ts

```diff
@@ -71,7 +72,7 @@ const getSharedLinkThunk = createAsyncThunk<string | void, GetLinkPayload, { sta
       if (!bucket) {
         trackShareLinkBucketIdUndefined({ email: bridgeUser });
         // close();
-        notificationsService.show({ text: i18n.get('error.shareLinkMissingBucket'), type: ToastType.Error });
+        notificationsService.show({ text: get('error.shareLinkMissingBucket'), type: ToastType.Error });
```

Comment:
remember to remove all  imported get in all files from 'app/i18n/services/i18n.service', it not gonna works.